lib/postal.lodash.js

```diff
@@ -539,12 +539,12 @@
             // First, add the SubscriptionDefinition to the channel list
             subs.push(subDef);
             // Next, add the SubscriptionDefinition to any relevant existing cache(s)
-            _.each(this.cache, function (list, cacheKey) {
+            _.each(_.keys(this.cache), function (cacheKey) {
                 if (cacheKey.substr(0, channelLen) === subDef.channel) {
                     getCacher(
-                    cacheKey.split(_config.cacheKeyDelimiter)[1], list, cacheKey)(subDef);
+                    cacheKey.split(_config.cacheKeyDelimiter)[1], this.cache, cacheKey)(subDef);
```

Comment:
@ifandelse kind of weird that we got `this.cache` equal to `undefined` on the `postal.lodash` build, and causing js error on `getCacher`, but it works fine on the normal build where `this.cache` return correctly an object.

did you see the same issue?